src/platform/forms-system/src/js/utilities/validations/index.js

```diff
@@ -55,6 +58,11 @@ export function isValidPartialDate(day, month, year) {
   return true;
 }
 
+export function isValidDate(day, month, year) {
+  const momentDate = moment({ day, month: parseInt(month, 10) - 1, year });
+  return momentDate.isValid();
```

Comment:
With Moment now considered to be in maintenance mode, would it be relatively easy to replace these moment functions with `date fns`? It already exists in vets-website's package.json, and they have a built in [isValid](https://date-fns.org/v2.16.1/docs/isValid) helper as well. 

Though there isn't any real official guidance on migrating vets-website from moment to date fns, I think it could be good to try and adopt it whenever possible.

Reply:
After review, I still had to use moment as the standard date library converts the invalid date to the next valid date. Thanks for the suggestion! Will keep this on my radar going forward.